.github/fabricbot.json

```diff
@@ -1,1470 +0,0 @@
-[
-  {
-    "taskSource": "fabricbot-config",
-    "taskType": "trigger",
-    "capabilityId": "IssueResponder",
-    "subCapability": "IssuesOnlyResponder",
-    "version": "1.0",
-    "config": {
-      "taskName": "Add untriaged label to new/reopened issues without a milestone",
```

Comment:
This task did not get converted, presumably because it uses the `removedFromMilestone` action which is not supported.